decentraland/unity-renderer
# fix: unhandled exception for gif non pow2

## What does this PR change?

This PR implements the [forked library Unity-GifDecoder](https://github.com/decentraland/Unity-GifDecoder) as a package.

[The corresponding PR with the fixed library, can be found here.
](https://github.com/decentraland/Unity-GifDecoder/pull/1)

...

## How to test the changes?



1. Launch the explorer
2. teleport to this coordinates, and see if there is any gif corruption exception in console /goto -56,-117

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
